app/src/main/java/com/example/kotlin/movieapp/custom_bindings/RecyclerViewBinding.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.kotlin.movieapp.custom_bindings
+
+import android.databinding.BindingAdapter
+import android.databinding.ObservableArrayList
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import com.example.kotlin.movieapp.adapter.MovieList
+import com.example.kotlin.movieapp.app.App
+import com.example.kotlin.movieapp.model.Movie
+
+object RecyclerViewBinding {
+    @JvmStatic
+    @BindingAdapter("bind:items")
+    fun setupItems(recyclerView: RecyclerView, items: ObservableArrayList<Movie>) {
+        var adapter = MovieList(items)
+        items.addOnListChangedCallback(adapter.onListChangeListener)
+        recyclerView.layoutManager = LinearLayoutManager(App.instance)
```

Comment:
instead of using the `App.instance` can't you use the `recyclerView.getContext()`?